cfpb/consumerfinance.gov
# Remove Haystack code, flag references, index file, and tests from PFC

Elasticsearch7 is live, so we can remove the Haystack code, files, tests and Flag from PFC.

## How to test this PR
- Run unittest-current and check 100% coverage
```
tox -e unittest-current paying_for_college
coverage report -m | egrep "Stmts|paying_for_college"
```

Check out the search page (Docker setup) and make sure search works as expected.
<http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/api/search-schools.json?q=Harvard>
